onnx/__init__.py

```diff
@@ -36,17 +36,22 @@
     "version_converter",
     # Proto classes
     "AttributeProto",
+    "DeviceConfigurationProto",
     "FunctionProto",
     "GraphProto",
     "MapProto",
     "ModelProto",
+    "NodeConfigurationProto",
```

Comment:
Is this proto only for device config? If so NodeDeviceConfig… for consistency with the DeviceConfig…?

Reply:
Renamed to NodeDeviceConfigurationProto